engine/src/uci/crazyara.h

```diff
@@ -186,6 +189,19 @@ class CrazyAra
      */
     void arena(istringstream &is);
 
+    /**
+    *
```

Comment:
Can you add some comments here, what these functions do?

Reply:
```suggestion
    * @brief mctsarena Alternative to the arena method which enables us to define two different models to use in the match and also define the mctsagent types to use.
     * @param is Input string representing both agent types and the number of games to play
     * @param modeldirectory1 name of the model directory of agent 1
     * @param modeldirectory2 name of the model directory of agent 2
```